cmd/fetchd/cmd/capricornmigrate.go

```diff
@@ -0,0 +1,489 @@
+package cmd
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"time"
+
+	"github.com/spf13/cobra"
+	tmjson "github.com/tendermint/tendermint/libs/json"
+	tmtypes "github.com/tendermint/tendermint/types"
+
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/errors"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil/types"
+	ibctransfer "github.com/cosmos/cosmos-sdk/x/ibc/applications/transfer/types"
+	ibchost "github.com/cosmos/cosmos-sdk/x/ibc/core/24-host"
+	ibccoretypes "github.com/cosmos/cosmos-sdk/x/ibc/core/types"
+	slashingtypes "github.com/cosmos/cosmos-sdk/x/slashing/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	"github.com/fetchai/fetchd/app"
+)
+
+const (
+	flagGenesisTime                    = "genesis-time"
+	flagInitialHeight                  = "initial-height"
+	flagConsensusBlockMaxBytes         = "consensus-block-max-bytes"
+	flagConsensusBlockMaxGas           = "consensus-block-max-gas"
+	flagFoundationAddress              = "foundation-address"
+	flagFoundationTokensToBurn         = "foundation-tokens-to-burn"
+	flagStakingParamsHistoricalEntries = "staking-historical-entries"
+	flagBridgeNewContractPath          = "bridge-new-contract-path"
+	flagMobixNewContractPath           = "mobix-new-contract-path"
+	flagNewSlashFractionDoubleSign     = "new-slash-fraction-double-sign"
+)
+
+const (
+	bridgeContractCodeID = 2
+	mobixContractCodeID  = 3
+)
+
+var (
+	// contractCodeIDsToRemove list the codeIDs of the contracts to be removed by the migration
+	contractCodeIDsToRemove = []string{
+		"1", // Unused bridge contract (effective one use codeID 2)
+		"4", // Reconciliation contract v0 (not needed anymore)
+		"5", // Reconciliation contract v1 (not needed anymore)
+	}
+)
+
+// AddCapricornMigrateCmd returns a command to migrate genesis to capricorn version.
+func AddCapricornMigrateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "capricorn-migrate <genesis-file> --bridge-new-contract-path /path/to/bridge.wasm --mobix-new-contract-path /path/to/mobix.wasm",
```

Comment:
What do you think about wrapping the required flags' values in angle brackets (i.e. `--bridge-new-contract-path </path/to/bridge.wasm>`)?